automation/src/test/scala/org/broadinstitute/dsde/test/api/SamApiSpec.scala

```diff
@@ -97,6 +99,19 @@ class SamApiSpec extends AnyFreeSpec with BillingFixtures with Matchers with Sca
   }
 
   "Sam" - {
+    "should return terms of services" in {
+      val anyUser: Credentials = UserPool.chooseAnyUser
+      val userAuthToken: AuthToken = anyUser.makeAuthToken()
+
+      val response = Sam.getRequest(Sam.url + s"tos/text")(userAuthToken)
+      val textFuture = Unmarshal(response.entity).to[String]
+
+      response.status shouldEqual StatusCodes.OK
+      whenReady(textFuture) { text =>
+        text should include("Terms as of February 12, 2020.")
+      }
+    }
+
```

Comment:
the ToS text should be available to users who haven't yet registered, since it's part of the registration flow. Could you add a test that checks anonymous access to /tos/text too?

Reply:
Yeah, I'll add an anonymous test-